app/controllers/v1/inventory_participants_controller.rb

```diff
@@ -18,8 +23,15 @@ def destroy
     render nothing: true
   end
 
-  protected
+  def all
+    district = inventory.district
+    participant_ids = inventory.members.pluck(:user_id)
+    @users = User.includes(:districts).where(districts: {id: district.id}).where.not(id: participant_ids)
+                 .reject { |u| u.role.to_sym == :network_partner }
+  end
+  authority_actions all: 'update'
```

Comment:
Are we sure about this? Even if the permissions for getting this list and updating a record are the same, I find this line confusing.